src/Constants.ts

```diff
@@ -1,10 +1,26 @@
 import os = require('os');
 import path = require('path');
 
-export const SFDX_SCANNER_PATH = path.join(os.homedir(), '.sfdx-scanner');
+export interface EnvOverridable {
+	getSfdxScannerPath(): string;
+}
+
+export class ProdOverrides implements EnvOverridable {
+	public getSfdxScannerPath(): string {
+		return path.join(os.homedir(), '.sfdx-scanner');
+	}
+}
+
+export class TestOverrides implements EnvOverridable {
+	public getSfdxScannerPath(): string {
+		return path.join(os.homedir(), '.sfdx-scanner-test');
+	}
+}
+
```

Comment:
I know we decided to move each of these constants to the respective module it is used - but I still like the accessibility factor of finding all the names in one place. Let me know if you feel strongly about moving it out of here.